app/src/androidTest/java/org/fossasia/pslab/MainActivityTest.java

```diff
@@ -38,8 +38,6 @@ public void checkingAllFragments() throws Throwable {
         // checking home fragment views
         onView(withId(R.id.img_device_status)).check(matches(isDisplayed()));
         onView(withId(R.id.tv_device_status)).check(matches(isDisplayed()));
-        onView(withId(R.id.tv_device_version)).check(matches(isDisplayed()));
```

Comment:
Please update your branch with development as this file was removed few days ago. It will create lot of conflicts in other PRs.

Reply:
@CloudyPadmal both the branches are showing 468 commits. Could u plz point to the commit which removed this test.

Reply:
Is your branch tracking the upstream? #909 removed these UI tests to increase build time as well as to make it easier with new UI layouts.

Reply:
These tests are not removed in this commit only ExampleInstrumentedTest and ExampleUnitTest are removed

Reply:
@Avjeet yes. Please remove this test file. Then we can merge this.

Reply:
@Avjeet no we'll do that in a different PR